modules/installation-initializing.adoc

```diff
@@ -44,8 +44,11 @@ ifeval::["{context}" == "installing-restricted-networks-aws-installer-provisione
 :aws:
 :restricted:
 endif::[]
-ifeval::["{context}" == "installing-azure-customizations"]
-:azure:
+ifeval::["{context}" == "installing-restricted-networks-aws"]
+:aws-restricted-network:
+endif::[]
+ifeval::["{context}" == "installing-azure-user-infra"]
```

Comment:
I don't think I've seen any record that insists it has to be this way, but my instinct is that if the ifeval conditions on lines 50 and 59 are the same, then I think that writers who follow behind you probably want them combined into one. I'm open to learning if there's an advantage with them separate.

Reply:
These files are similar, but have different content.

Reply:
I made this change. Nothing seems broken. :-)